### ISSUE TITLE
Tutorial section

### ISSUE THREAD

**Issue by pathumego** [trufont/trufont.github.io#7]
As per [trufont/trufont#223](https://github.com/trufont/trufont/issues/223) We want to start a tutorial section. How do we do it?

**Comment by adrientetar** [comment]
Go ahead making something and then I'll look at it.

**Comment by madig** [comment]
I like the format of Glyphs's tutorials, like here: https://glyphsapp.com/tutorials/sketching

How about we walk people through making a simple font:
1. Using the drawing tools to make outlines for `n` and `o` some other glyphs. This explains how to work with corner and smooth points and how to use the tools.
2. Explain the component system and make some composite glyphs with it
3. Work people through the spacing dialog
4. Go through the font properties dialog to set basic meta data
5. Generate the font

The focus would be on explaining the tools and workflow, not designing glyphs.

**Comment by madig** [comment]
Oh, but I'd wait for trufont/trufont#176 before starting a tutorial, as those are rather basic and necessary tools.

**Comment by pathumego** [comment]
@madig wouldn hurt it to start planing. @davelab6 suggested working on something similar to DesingwithFontforge.com. We can simply port the exisiting content and rewrite to work with trufont

**Comment by adrientetar** [comment]
Yea I don't want a teach me type design thing, we have to stay focused on how to use the application. Examples don't hurt, but stay in line. I like that Glyphs documentation has smaller tutorials that are focused.

**Comment by madig** [comment]
I just found out about birdfont. They have tutorials on their page: https://birdfont.org/doku/doku.php/tutorials -- the focus is on tools.

**Comment by pathumego** [comment]
@adrientetar I agree. And I think Dave wrote video castas are better in this stage, on a different issue.

**Comment by davelab6** [comment]
Oops, it was in the wrong issue. Here it is :)

---

I think this is a bit premature. 

For those users who are eager to start, though, then I suggest accumulating "tips and tricks" for how **YOU** personally do the things that you frequently do. Screencasts are probably ideal for this, along with something that shows keypresses and mouseclicks on screen. 

Some ideas:

Mac: 
- http://silverbackapp.com
- https://www.boinx.com/mousepose/overview/ 

Windows: 
- https://lesliesrussell.wordpress.com/2006/07/21/mousepose-for-windows/
- http://wisdom-soft.com/sh/sh_capture.htm

GNU: 

I'm not sure what the best stuff is. I am sad that @felipesanches never really shipped a libre version of  for GNOME3, and don't know what libre tools to recommend to make narrated screencasts (if not video-narrated as silverback offers) but I think an easy proprietary solution is Google Hangouts on Air. 
- https://www.google.com/search?q=key-mon+github
- http://www.maartenbaert.be/simplescreenrecorder/ 
- https://obsproject.com/
- https://apps.ubuntu.com/cat/applications/kazam/

**Comment by davelab6** [comment]
https://github.com/davelab6/Python-GUI-Programming-Cookbook-Second-Edition will be useful soon :)

Also https://github.com/driscollis/wxPython_recipes_book_code

**Comment by madig** [comment]
Trufont is going to be rewritten in tk? 😁
